django_freeradius/base/models.py

```diff
@@ -819,3 +820,36 @@ def _remove_files(self):
         path = getattr(self, strategy_filemap.get(self.strategy)).path
         if os.path.isfile(path):
             os.remove(path)
+
+
+class AbstractRadiusToken(AutoUsernameMixin, models.Model):
+    key = models.CharField(_("Key"), max_length=40, primary_key=True)
+
+    username = models.CharField(verbose_name=_('username'),
```

Comment:
why do we need this?
this looks like it shouldn't be here. We only need the key.

Reply:
PS keep the quote style consistent

Reply:
@nemesisdesign  do we need both the username and user field in this model like other models, or should I just drop the username field?

Reply:
drop it, is not needed